packages/plugin-ext/src/hosted/node/plugin-reader.ts

```diff
@@ -50,8 +50,13 @@ export class HostedPluginReader implements BackendApplicationContribution {
 
             const localPath = this.pluginsIdsFiles.get(pluginId);
             if (localPath) {
-                res.sendFile(filePath, { root: localPath }, (error: Error) => {
-                    res.status(404).send(`No such file for plugin with id '${escape_html(pluginId)}'.`);
+                res.sendFile(filePath, { root: localPath }, e => {
+                    if (e) {
```

Comment:
Is this check necessary? Would e ever be null if the error function i called? Also, if the error function is called without an "e", should we not handle that as well (i.e. log it and send a response "500" maybe?)

Reply:
it is a result callback, error is only defined if something went wrong, if response was successful then it is undefined